src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -328,14 +341,15 @@ class ActivityFeed extends React.Component<Props, State> {
 
     const tokenTxHistory = history.filter(({ tranType }) => tranType !== 'collectible');
     const collectibleTxHistory = history.filter(({ tranType }) => tranType === 'collectible');
-    // merging BCX and OpenSea transaction data by hash
-    const mergedCollectibleTxHistory = values(
-      merge({}, keyBy(collectibleTxHistory, 'hash'), keyBy(collectiblesHistory, 'hash')));
+
+    // extending OpenSea transaction data with BCX data
+    const extendedCollectiblesTransactionData =
```

Comment:
#pickyJegor
Maybe we should call it like `const collectiblesHistory = this.mapOpenSeaAndBCXTransactionsHistory(openSeaTxHistory, bcxCollectiblesTxHistory)`?
and choose one option, either it's 'collectiblesHistory' or 'collectibleHistory' =)